{"TimeStamp":"2025-01-23T21:21:33.290147284Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:21:33 debian systemd[25573]: Listening on GnuPG cryptographic agent (ssh-agent emulation).\nJan 23 21:21:33 debian systemd[25573]: Listening on GnuPG cryptographic agent and passphrase cache.\nJan 23 21:21:33 debian systemd[25573]: Reached target Sockets.\nJan 23 21:21:33 debian systemd[25573]: Reached target Basic System.\nJan 23 21:21:33 debian systemd[1]: Started User Manager for UID 1001.\nJan 23 21:21:33 debian systemd[1]: Started Session 200 of user blankcanvas.\nJan 23 21:21:33 debian systemd[25573]: Reached target Main User Target.\nJan 23 21:21:33 debian systemd[25573]: Startup finished in 78ms.\nJan 23 21:21:33 debian discern-proc-sorcerer[24076]: #033[31mERRO#033[0m[21:21:33] [Proc] error saving proc-cpu info for pid 25573: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-proc-sorcerer[24076]: #033[31mERRO#033[0m[21:21:33] [Proc] error saving proc-cpu info for pid 24076: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-proc-sorcerer[24076]: #033[31mERRO#033[0m[21:21:33] [Proc] error saving proc-cpu info for pid 25549: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-proc-sorcerer[24076]: #033[31mERRO#033[0m[21:21:33] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-proc-sorcerer[24076]: #033[31mERRO#033[0m[21:21:33] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:33] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-proc-sorcerer[24076]: #033[31mERRO#033[0m[21:21:33] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-proc-sorcerer[24076]: #033[31mERRO#033[0m[21:21:33] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-proc-sorcerer[24076]: #033[31mERRO#033[0m[21:21:33] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-proc-sorcerer[24076]: #033[31mERRO#033[0m[21:21:33] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-proc-sorcerer[24076]: #033[31mERRO#033[0m[21:21:33] [Proc] error saving proc-mem info for pid 24076: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:33] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:33] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:33] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:33] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:21:33] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:21:33.292236199Z", "Location":"/var/log/auth.log", "Content":"Jan 23 21:21:33 debian sshd[25549]: pam_unix(sshd:session): session opened for user blankcanvas(uid=1001) by (uid=0)\nJan 23 21:21:33 debian systemd-logind[388]: New session 200 of user blankcanvas.\nJan 23 21:21:33 debian systemd: pam_unix(systemd-user:session): session opened for user blankcanvas(uid=1001) by (uid=0)\nJan 23 21:21:33 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/mkdir -p /var/log/ttylog/\nJan 23 21:21:33 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:21:33 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:33 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/[ -e /tmp/count.blankcanvas ]\nJan 23 21:21:33 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:21:33 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:33 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/cat /tmp/count.blankcanvas\nJan 23 21:21:33 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:21:33 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:33 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/tee /tmp/count.blankcanvas\nJan 23 21:21:33 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:21:33 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:33 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/touch /var/log/ttylog/ttylog.cache.infra.real.dnsmitm.blankcanvas.blankcanvas.160.trace\nJan 23 21:21:33 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:21:33 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:33 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/chmod ugo+rw /var/log/ttylog/ttylog.cache.infra.real.dnsmitm.blankcanvas.blankcanvas.160.trace\nJan 23 21:21:33 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:21:33 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:33 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/rm -rf /var/log/discern/data/bash-data.txt /var/log/discern/data/cpu-load-data.txt /var/log/discern/data/file-data.txt /var/log/discern/data/logs-data.txt /var/log/discern/data/network-data.txt /var/log/discern/data/proc-cpu-data.txt /var/log/discern/data/proc-mem-data.txt /var/log/discern/data/proc-new-data.txt\nJan 23 21:21:33 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:21:33 debian sudo: pam_unix(sudo:session): session closed for user root\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:21:33.299489Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:21:33 debian systemd[1]: Started User Manager for UID 1001.\nJan 23 21:21:33 debian systemd[1]: Started Session 200 of user blankcanvas.\nJan 23 21:21:33 debian systemd[25573]: Reached target Main User Target.\nJan 23 21:21:33 debian systemd[25573]: Startup finished in 78ms.\nJan 23 21:21:33 debian discern-proc-sorcerer[24076]: #033[31mERRO#033[0m[21:21:33] [Proc] error saving proc-cpu info for pid 25573: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-proc-sorcerer[24076]: #033[31mERRO#033[0m[21:21:33] [Proc] error saving proc-cpu info for pid 24076: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-proc-sorcerer[24076]: #033[31mERRO#033[0m[21:21:33] [Proc] error saving proc-cpu info for pid 25549: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-proc-sorcerer[24076]: #033[31mERRO#033[0m[21:21:33] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-proc-sorcerer[24076]: #033[31mERRO#033[0m[21:21:33] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:33] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-proc-sorcerer[24076]: #033[31mERRO#033[0m[21:21:33] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-proc-sorcerer[24076]: #033[31mERRO#033[0m[21:21:33] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-proc-sorcerer[24076]: #033[31mERRO#033[0m[21:21:33] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-proc-sorcerer[24076]: #033[31mERRO#033[0m[21:21:33] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-proc-sorcerer[24076]: #033[31mERRO#033[0m[21:21:33] [Proc] error saving proc-mem info for pid 24076: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:33] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:33] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:33] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:33] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:21:33] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 23 21:21:33 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:21:33] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:33] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:21:33] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian systemd[1]: session-200.scope: Succeeded.\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:21:38.322627443Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:21:33 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:33] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:21:33] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian systemd[1]: session-200.scope: Succeeded.\nJan 23 21:21:33 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:21:33] [Logs] error saving /var/log/daemon.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:33 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:21:33] [Logs] Sending has completed\nJan 23 21:21:33 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:33] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:34 debian discern-cpu-sorcerer[2952]: #033[31mERRO#033[0m[21:21:34] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:35 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:35] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:35 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 378.\nJan 23 21:21:35 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:21:35 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:21:35 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:21:35 debian discern-proc-sorcerer[24076]: #033[36mINFO#033[0m[21:21:35] [Proc] stripping info for: 25574\nJan 23 21:21:35 debian discern-proc-sorcerer[24076]: #033[36mINFO#033[0m[21:21:35] [Proc] stripping info for: 25573\nJan 23 21:21:35 debian discern-proc-sorcerer[24076]: #033[31mERRO#033[0m[21:21:35] [Proc] error saving proc-new info for pid 25574: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:35 debian discern-proc-sorcerer[24076]: #033[31mERRO#033[0m[21:21:35] [Proc] error saving proc-new info for pid 25573: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:35 debian discern-proc-sorcerer[24076]: #033[36mINFO#033[0m[21:21:35] [Proc] successfully saved data to the fusion core in normal loop\nJan 23 21:21:38 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:21:38] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:21:38.325078330Z", "Location":"/var/log/auth.log", "Content":"Jan 23 21:21:33 debian sshd[25591]: Disconnected from user blankcanvas 192.168.254.2 port 41562\nJan 23 21:21:33 debian sshd[25549]: pam_unix(sshd:session): session closed for user blankcanvas\nJan 23 21:21:33 debian systemd-logind[388]: Session 200 logged out. Waiting for processes to exit.\nJan 23 21:21:33 debian systemd-logind[388]: Removed session 200.\nJan 23 21:21:35 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/open.bt\nJan 23 21:21:35 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/fork.bt\nJan 23 21:21:35 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/kill.bt\nJan 23 21:21:35 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:21:35 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:21:35 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:35 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:35 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:21:35 debian sudo: pam_unix(sudo:session): session closed for user root\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:21:38.327359706Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:21:33 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:21:33] [Logs] Sending has completed\nJan 23 21:21:33 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:33] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:34 debian discern-cpu-sorcerer[2952]: #033[31mERRO#033[0m[21:21:34] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:35 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:35] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:35 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 378.\nJan 23 21:21:35 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:21:35 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:21:35 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:21:35 debian discern-proc-sorcerer[24076]: #033[36mINFO#033[0m[21:21:35] [Proc] stripping info for: 25574\nJan 23 21:21:35 debian discern-proc-sorcerer[24076]: #033[36mINFO#033[0m[21:21:35] [Proc] stripping info for: 25573\nJan 23 21:21:35 debian discern-proc-sorcerer[24076]: #033[31mERRO#033[0m[21:21:35] [Proc] error saving proc-new info for pid 25574: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:35 debian discern-proc-sorcerer[24076]: #033[31mERRO#033[0m[21:21:35] [Proc] error saving proc-new info for pid 25573: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:35 debian discern-proc-sorcerer[24076]: #033[36mINFO#033[0m[21:21:35] [Proc] successfully saved data to the fusion core in normal loop\nJan 23 21:21:38 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:21:38] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 23 21:21:38 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:21:38] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:38 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:21:38] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:21:43.345470786Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:21:41 debian systemd[1]: session-202.scope: Succeeded.\nJan 23 21:21:41 debian discern-proc-sorcerer[25710]: #033[36mINFO#033[0m[21:21:41] [Proc] hello world from Proc scraper\nJan 23 21:21:41 debian discern-proc-sorcerer[25710]: #033[36mINFO#033[0m[21:21:41] [Proc] connected to fusion core\nJan 23 21:21:41 debian discern-proc-sorcerer[25710]: #033[36mINFO#033[0m[21:21:41] [Proc] made intermediate data file folder\nJan 23 21:21:41 debian discern-proc-sorcerer[25710]: #033[36mINFO#033[0m[21:21:41] [Proc] successfully saved data to the fusion core in normal loop\nJan 23 21:21:41 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:41] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:41 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:41] [Proc] error saving proc-cpu info for pid 25710: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:41 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:41] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:41 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:41] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:41 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:41] [Proc] error saving proc-cpu info for pid 25573: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:41 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:41] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:41 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:41] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:41 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:41] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:41 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:41] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:41 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:41] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:41 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:41] [Proc] error saving proc-mem info for pid 9394: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:42 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:42] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:42 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:42] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:42 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 380.\nJan 23 21:21:42 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:21:42 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:21:42 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:21:42 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:42] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:43 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:21:43] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:21:43.348038274Z", "Location":"/var/log/auth.log", "Content":"Jan 23 21:21:41 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:41 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/systemctl restart discern-proc\nJan 23 21:21:41 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:21:41 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:41 debian sshd[25676]: Received disconnect from 192.168.254.2 port 41578:11: disconnected by user\nJan 23 21:21:41 debian sshd[25676]: Disconnected from user blankcanvas 192.168.254.2 port 41578\nJan 23 21:21:41 debian sshd[25670]: pam_unix(sshd:session): session closed for user blankcanvas\nJan 23 21:21:41 debian systemd-logind[388]: Session 202 logged out. Waiting for processes to exit.\nJan 23 21:21:41 debian systemd-logind[388]: Removed session 202.\nJan 23 21:21:42 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/kill.bt\nJan 23 21:21:42 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/fork.bt\nJan 23 21:21:42 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/execveat.bt\nJan 23 21:21:42 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/open.bt\nJan 23 21:21:42 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/openat.bt\nJan 23 21:21:42 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:21:42 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:21:42 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:21:42 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:21:42 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:21:42 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:42 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:42 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:42 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:42 debian sudo: pam_unix(sudo:session): session closed for user root\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:21:43.354042656Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:21:41 debian discern-proc-sorcerer[25710]: #033[36mINFO#033[0m[21:21:41] [Proc] connected to fusion core\nJan 23 21:21:41 debian discern-proc-sorcerer[25710]: #033[36mINFO#033[0m[21:21:41] [Proc] made intermediate data file folder\nJan 23 21:21:41 debian discern-proc-sorcerer[25710]: #033[36mINFO#033[0m[21:21:41] [Proc] successfully saved data to the fusion core in normal loop\nJan 23 21:21:41 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:41] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:41 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:41] [Proc] error saving proc-cpu info for pid 25710: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:41 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:41] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:41 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:41] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:41 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:41] [Proc] error saving proc-cpu info for pid 25573: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:41 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:41] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:41 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:41] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:41 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:41] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:41 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:41] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:41 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:41] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:41 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:41] [Proc] error saving proc-mem info for pid 9394: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:42 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:42] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:42 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:42] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:42 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 380.\nJan 23 21:21:42 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:21:42 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:21:42 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:21:42 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:42] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:43 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:21:43] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 23 21:21:43 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:21:43] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:43 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:21:43] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:21:48.372038154Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:21:45 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:45] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:45 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:45] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:45 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:45] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:45 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:45] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:45 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:45 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 381.\nJan 23 21:21:45 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:21:45 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:21:45 debian systemd[1]: session-204.scope: Succeeded.\nJan 23 21:21:45 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:21:46 debian discern-proc-sorcerer[25710]: #033[36mINFO#033[0m[21:21:46] [Proc] successfully saved data to the fusion core in normal loop\nJan 23 21:21:46 debian discern-cpu-sorcerer[2952]: #033[31mERRO#033[0m[21:21:46] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:46 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:46] [Proc] error saving proc-cpu info for pid 25710: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:46 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:46] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:46 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:46] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:46 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:46] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:46 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:46] [Proc] error saving proc-cpu info for pid 25573: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:46 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:46] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:46 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:46] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:46 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:46] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:46 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:46] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:46 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:46] [Proc] error saving proc-mem info for pid 25710: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:46 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:46] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:48 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:21:48] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:21:48.374953695Z", "Location":"/var/log/auth.log", "Content":"Jan 23 21:21:45 debian systemd-logind[388]: New session 204 of user blankcanvas.\nJan 23 21:21:45 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/mkdir -p /var/log/ttylog/\nJan 23 21:21:45 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:21:45 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:45 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/[ -e /tmp/count.blankcanvas ]\nJan 23 21:21:45 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:21:45 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:45 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/cat /tmp/count.blankcanvas\nJan 23 21:21:45 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:21:45 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:45 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/tee /tmp/count.blankcanvas\nJan 23 21:21:45 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:21:45 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:45 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/touch /var/log/ttylog/ttylog.cache.infra.real.dnsmitm.blankcanvas.blankcanvas.163.trace\nJan 23 21:21:45 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:21:45 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:45 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/chmod ugo+rw /var/log/ttylog/ttylog.cache.infra.real.dnsmitm.blankcanvas.blankcanvas.163.trace\nJan 23 21:21:45 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:21:45 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:45 debian sshd[25784]: Received disconnect from 192.168.254.2 port 60660:11: disconnected by user\nJan 23 21:21:45 debian sshd[25784]: Disconnected from user blankcanvas 192.168.254.2 port 60660\nJan 23 21:21:45 debian sshd[25778]: pam_unix(sshd:session): session closed for user blankcanvas\nJan 23 21:21:45 debian systemd-logind[388]: Session 204 logged out. Waiting for processes to exit.\nJan 23 21:21:45 debian systemd-logind[388]: Removed session 204.\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:21:48.381061348Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:21:45 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:45] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:45 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:45] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:45 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:45 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 381.\nJan 23 21:21:45 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:21:45 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:21:45 debian systemd[1]: session-204.scope: Succeeded.\nJan 23 21:21:45 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:21:46 debian discern-proc-sorcerer[25710]: #033[36mINFO#033[0m[21:21:46] [Proc] successfully saved data to the fusion core in normal loop\nJan 23 21:21:46 debian discern-cpu-sorcerer[2952]: #033[31mERRO#033[0m[21:21:46] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:46 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:46] [Proc] error saving proc-cpu info for pid 25710: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:46 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:46] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:46 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:46] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:46 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:46] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:46 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:46] [Proc] error saving proc-cpu info for pid 25573: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:46 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:46] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:46 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:46] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:46 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:46] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:46 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:46] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:46 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:21:46] [Proc] error saving proc-mem info for pid 25710: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:46 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:46] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:48 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:21:48] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 23 21:21:48 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:21:48] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:48 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:21:48] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:21:53.398233628Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:21:52 debian discern-cpu-sorcerer[2952]: #033[31mERRO#033[0m[21:21:52] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:52 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:52] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:52 debian systemd[1]: Started Session 205 of user blankcanvas.\nJan 23 21:21:52 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:52] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:52 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:52] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:52 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:52] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:52 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:52] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:52 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:52] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:52 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:52] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:52 debian systemd[1]: session-205.scope: Succeeded.\nJan 23 21:21:52 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:52] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:52 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:52] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:52 debian systemd[1]: Started Session 206 of user blankcanvas.\nJan 23 21:21:53 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:53] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:53 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:53] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:53 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:53] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:53 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:53] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:53 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:53] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:53 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:53] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:53 debian systemd[1]: session-206.scope: Succeeded.\nJan 23 21:21:53 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:53] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:53 debian systemd[1]: Started Session 207 of user blankcanvas.\nJan 23 21:21:53 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:53] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:53 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:21:53] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:21:53.400823547Z", "Location":"/var/log/auth.log", "Content":"Jan 23 21:21:53 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:53 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/cat /tmp/count.blankcanvas\nJan 23 21:21:53 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:21:53 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:53 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/tee /tmp/count.blankcanvas\nJan 23 21:21:53 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:21:53 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:53 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/touch /var/log/ttylog/ttylog.cache.infra.real.dnsmitm.blankcanvas.blankcanvas.165.trace\nJan 23 21:21:53 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:21:53 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:53 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/chmod ugo+rw /var/log/ttylog/ttylog.cache.infra.real.dnsmitm.blankcanvas.blankcanvas.165.trace\nJan 23 21:21:53 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:21:53 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:53 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/cp /tmp/named.conf.options-with-dnssec /etc/bind/named.conf.options-with-dnssec\nJan 23 21:21:53 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:21:53 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:53 debian sshd[25925]: Received disconnect from 192.168.254.2 port 40658:11: disconnected by user\nJan 23 21:21:53 debian sshd[25925]: Disconnected from user blankcanvas 192.168.254.2 port 40658\nJan 23 21:21:53 debian sshd[25919]: pam_unix(sshd:session): session closed for user blankcanvas\nJan 23 21:21:53 debian systemd-logind[388]: Session 206 logged out. Waiting for processes to exit.\nJan 23 21:21:53 debian systemd-logind[388]: Removed session 206.\nJan 23 21:21:53 debian sshd[25960]: Accepted publickey for blankcanvas from 192.168.254.2 port 40666 ssh2: RSA SHA256:3I/WEXJMnChkvSdfEeoZ6YTUz2OrccqDruFNY41qR5o\nJan 23 21:21:53 debian sshd[25960]: pam_unix(sshd:session): session opened for user blankcanvas(uid=1001) by (uid=0)\nJan 23 21:21:53 debian systemd-logind[388]: New session 207 of user blankcanvas.\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:21:53.406882239Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:21:52 debian systemd[1]: Started Session 205 of user blankcanvas.\nJan 23 21:21:52 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:52] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:52 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:52] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:52 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:52] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:52 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:52] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:52 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:52] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:52 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:52] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:52 debian systemd[1]: session-205.scope: Succeeded.\nJan 23 21:21:52 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:52] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:52 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:52] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:52 debian systemd[1]: Started Session 206 of user blankcanvas.\nJan 23 21:21:53 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:53] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:53 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:53] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:53 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:53] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:53 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:53] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:53 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:21:53] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:53 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:53] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:53 debian systemd[1]: session-206.scope: Succeeded.\nJan 23 21:21:53 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:53] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:53 debian systemd[1]: Started Session 207 of user blankcanvas.\nJan 23 21:21:53 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:53] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:53 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:21:53] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 23 21:21:53 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:21:53] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:53 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:21:53] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:21:58.428615796Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:21:58 debian named[26230]: network unreachable resolving './DNSKEY/IN': 2001:503:ba3e::2:30#53\nJan 23 21:21:58 debian named[26230]: network unreachable resolving './DNSKEY/IN': 2001:500:a8::e#53\nJan 23 21:21:58 debian named[26230]: network unreachable resolving './DNSKEY/IN': 2001:500:9f::42#53\nJan 23 21:21:58 debian named[26230]: network unreachable resolving './DNSKEY/IN': 2001:500:2f::f#53\nJan 23 21:21:58 debian named[26230]: network unreachable resolving './DNSKEY/IN': 2001:7fe::53#53\nJan 23 21:21:58 debian named[26230]: network unreachable resolving './DNSKEY/IN': 2001:7fd::1#53\nJan 23 21:21:58 debian named[26230]: network unreachable resolving './DNSKEY/IN': 2001:500:2d::d#53\nJan 23 21:21:58 debian named[26230]: network unreachable resolving './DNSKEY/IN': 2801:1b8:10::b#53\nJan 23 21:21:58 debian named[26230]: network unreachable resolving './DNSKEY/IN': 2001:500:1::53#53\nJan 23 21:21:58 debian named[26230]: network unreachable resolving './DNSKEY/IN': 2001:500:12::d0d#53\nJan 23 21:21:58 debian named[26230]: timed out resolving './DNSKEY/IN': 10.1.2.3#53\nJan 23 21:21:58 debian named[26230]: network unreachable resolving './DNSKEY/IN': 2801:1b8:10::b#53\nJan 23 21:21:58 debian named[26230]: network unreachable resolving './DNSKEY/IN': 2001:dc3::35#53\nJan 23 21:21:58 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:58] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:58 debian named[26230]: managed-keys-zone: Initializing automatic trust anchor management for zone '.'; DNSKEY ID 20326 is now trusted, waiving the normal 30-day waiting period.\nJan 23 21:21:58 debian named[26230]: managed-keys-zone: Initializing automatic trust anchor management for zone '.'; DNSKEY ID 38696 is now trusted, waiving the normal 30-day waiting period.\nJan 23 21:21:58 debian named[26230]: resolver priming query complete\nJan 23 21:21:58 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:58] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:58 debian discern-cpu-sorcerer[2952]: #033[31mERRO#033[0m[21:21:58] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:58 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 385.\nJan 23 21:21:58 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:21:58 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:21:58 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:21:58 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:21:58] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:21:58.431771867Z", "Location":"/var/log/auth.log", "Content":"Jan 23 21:21:56 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:56 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/systemctl restart named\nJan 23 21:21:56 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:21:56 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:56 debian sshd[26191]: Received disconnect from 192.168.254.2 port 40698:11: disconnected by user\nJan 23 21:21:56 debian sshd[26191]: Disconnected from user blankcanvas 192.168.254.2 port 40698\nJan 23 21:21:56 debian sshd[26185]: pam_unix(sshd:session): session closed for user blankcanvas\nJan 23 21:21:56 debian systemd-logind[388]: Session 212 logged out. Waiting for processes to exit.\nJan 23 21:21:56 debian systemd-logind[388]: Removed session 212.\nJan 23 21:21:58 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/kill.bt\nJan 23 21:21:58 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/open.bt\nJan 23 21:21:58 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/execveat.bt\nJan 23 21:21:58 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/fork.bt\nJan 23 21:21:58 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:21:58 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:21:58 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:21:58 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/openat.bt\nJan 23 21:21:58 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:58 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:21:58 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:58 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:21:58 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:58 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:21:58 debian sudo: pam_unix(sudo:session): session closed for user root\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:21:58.437624568Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:21:58 debian named[26230]: network unreachable resolving './DNSKEY/IN': 2001:500:9f::42#53\nJan 23 21:21:58 debian named[26230]: network unreachable resolving './DNSKEY/IN': 2001:500:2f::f#53\nJan 23 21:21:58 debian named[26230]: network unreachable resolving './DNSKEY/IN': 2001:7fe::53#53\nJan 23 21:21:58 debian named[26230]: network unreachable resolving './DNSKEY/IN': 2001:7fd::1#53\nJan 23 21:21:58 debian named[26230]: network unreachable resolving './DNSKEY/IN': 2001:500:2d::d#53\nJan 23 21:21:58 debian named[26230]: network unreachable resolving './DNSKEY/IN': 2801:1b8:10::b#53\nJan 23 21:21:58 debian named[26230]: network unreachable resolving './DNSKEY/IN': 2001:500:1::53#53\nJan 23 21:21:58 debian named[26230]: network unreachable resolving './DNSKEY/IN': 2001:500:12::d0d#53\nJan 23 21:21:58 debian named[26230]: timed out resolving './DNSKEY/IN': 10.1.2.3#53\nJan 23 21:21:58 debian named[26230]: network unreachable resolving './DNSKEY/IN': 2801:1b8:10::b#53\nJan 23 21:21:58 debian named[26230]: network unreachable resolving './DNSKEY/IN': 2001:dc3::35#53\nJan 23 21:21:58 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:58] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:58 debian named[26230]: managed-keys-zone: Initializing automatic trust anchor management for zone '.'; DNSKEY ID 20326 is now trusted, waiving the normal 30-day waiting period.\nJan 23 21:21:58 debian named[26230]: managed-keys-zone: Initializing automatic trust anchor management for zone '.'; DNSKEY ID 38696 is now trusted, waiving the normal 30-day waiting period.\nJan 23 21:21:58 debian named[26230]: resolver priming query complete\nJan 23 21:21:58 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:21:58] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:58 debian discern-cpu-sorcerer[2952]: #033[31mERRO#033[0m[21:21:58] [CPU] error saving cpu info to fusion core: client error saving cpu event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:58 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 385.\nJan 23 21:21:58 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:21:58 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:21:58 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:21:58 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:21:58] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 23 21:21:58 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:21:58] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:58 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:21:58] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:22:03.452921906Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:21:58 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:21:58] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:58 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:21:58] [Logs] error saving /var/log/daemon.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:21:58 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:21:58] [Logs] Sending has completed\nJan 23 21:22:01 debian discern-proc-sorcerer[25710]: #033[36mINFO#033[0m[21:22:01] [Proc] stripping info for: 26230\nJan 23 21:22:01 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:01] [Proc] error saving proc-new info for pid 26230: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:01 debian discern-proc-sorcerer[25710]: #033[36mINFO#033[0m[21:22:01] [Proc] successfully saved data to the fusion core in normal loop\nJan 23 21:22:01 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 386.\nJan 23 21:22:01 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:01 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:01 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:22:01 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:01] [Proc] error saving proc-cpu info for pid 25710: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:01 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:01] [Proc] error saving proc-cpu info for pid 26230: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:01 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:01] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:01 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:01] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:01 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:01] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:01 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:01] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:01 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:01] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:01 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:01] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:01 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:01] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:01 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:01] [Proc] error saving proc-mem info for pid 26230: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:03 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:03] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:22:03.459658174Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:21:58 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:21:58] [Logs] Sending has completed\nJan 23 21:22:01 debian discern-proc-sorcerer[25710]: #033[36mINFO#033[0m[21:22:01] [Proc] stripping info for: 26230\nJan 23 21:22:01 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:01] [Proc] error saving proc-new info for pid 26230: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:01 debian discern-proc-sorcerer[25710]: #033[36mINFO#033[0m[21:22:01] [Proc] successfully saved data to the fusion core in normal loop\nJan 23 21:22:01 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 386.\nJan 23 21:22:01 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:01 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:01 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:22:01 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:01] [Proc] error saving proc-cpu info for pid 25710: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:01 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:01] [Proc] error saving proc-cpu info for pid 26230: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:01 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:01] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:01 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:01] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:01 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:01] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:01 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:01] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:01 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:01] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:01 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:01] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:01 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:01] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:01 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:01] [Proc] error saving proc-mem info for pid 26230: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:03 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:03] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 23 21:22:03 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:22:03] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:22:08.476349499Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:22:06 debian systemd[1]: run-user-1001.mount: Succeeded.\nJan 23 21:22:06 debian systemd[1]: user-runtime-dir@1001.service: Succeeded.\nJan 23 21:22:06 debian systemd[1]: Stopped User Runtime Directory /run/user/1001.\nJan 23 21:22:06 debian systemd[1]: Removed slice User Slice of UID 1001.\nJan 23 21:22:07 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:07] [Proc] error saving proc-cpu info for pid 25710: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:07 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:07] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:07 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:07] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:07 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:07] [Proc] error saving proc-cpu info for pid 26230: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:07 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:07] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:07 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:07] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:07 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:07] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:07 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:07] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:07 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:07] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:07 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:07] [Proc] error saving proc-mem info for pid 26230: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:08 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 388.\nJan 23 21:22:08 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:08 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:08 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:22:08 debian named[26230]: timed out resolving 'com/DS/IN': 10.1.2.3#53\nJan 23 21:22:08 debian named[26230]: network unreachable resolving 'com/DS/IN': 2001:500:a8::e#53\nJan 23 21:22:08 debian named[26230]: network unreachable resolving 'com/DS/IN': 2001:500:12::d0d#53\nJan 23 21:22:08 debian named[26230]: network unreachable resolving 'com/DS/IN': 2001:500:9f::42#53\nJan 23 21:22:08 debian named[26230]: network unreachable resolving 'com/DS/IN': 2001:500:1::53#53\nJan 23 21:22:08 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:08] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:22:08.478887007Z", "Location":"/var/log/auth.log", "Content":"Jan 23 21:22:04 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/openat.bt\nJan 23 21:22:04 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/openat2.bt\nJan 23 21:22:04 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/open.bt\nJan 23 21:22:04 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:04 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/fork.bt\nJan 23 21:22:04 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/execveat.bt\nJan 23 21:22:04 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:04 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:04 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:04 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:04 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:04 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:04 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:04 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:04 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:04 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:04 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:06 debian systemd: pam_unix(systemd-user:session): session closed for user blankcanvas\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:22:08.485057270Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:22:06 debian systemd[1]: Stopped User Runtime Directory /run/user/1001.\nJan 23 21:22:06 debian systemd[1]: Removed slice User Slice of UID 1001.\nJan 23 21:22:07 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:07] [Proc] error saving proc-cpu info for pid 25710: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:07 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:07] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:07 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:07] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:07 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:07] [Proc] error saving proc-cpu info for pid 26230: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:07 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:07] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:07 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:07] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:07 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:07] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:07 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:07] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:07 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:07] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:07 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:07] [Proc] error saving proc-mem info for pid 26230: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:08 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 388.\nJan 23 21:22:08 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:08 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:08 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:22:08 debian named[26230]: timed out resolving 'com/DS/IN': 10.1.2.3#53\nJan 23 21:22:08 debian named[26230]: network unreachable resolving 'com/DS/IN': 2001:500:a8::e#53\nJan 23 21:22:08 debian named[26230]: network unreachable resolving 'com/DS/IN': 2001:500:12::d0d#53\nJan 23 21:22:08 debian named[26230]: network unreachable resolving 'com/DS/IN': 2001:500:9f::42#53\nJan 23 21:22:08 debian named[26230]: network unreachable resolving 'com/DS/IN': 2001:500:1::53#53\nJan 23 21:22:08 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:08] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 23 21:22:08 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:22:08] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:08 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:22:08] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:22:13.505855216Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:22:12 debian discern-file-sorcerer[2764]: #033[31mERRO#033[0m[21:22:12] [File] error saving fs event: error saving fs event: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:12 debian systemd[1]: session-213.scope: Succeeded.\nJan 23 21:22:12 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:12] [Proc] error stipping top cpu pid 26347: error loading /proc/26347/status: open /proc/26347/status: no such file or directory\nJan 23 21:22:12 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:12] [Proc] error saving proc-cpu info for pid 26350: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:22:12 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:12] [Proc] error saving proc-cpu info for pid 25710: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:22:12 debian discern-proc-sorcerer[25710]: panic: runtime error: invalid memory address or nil pointer dereference\nJan 23 21:22:12 debian discern-proc-sorcerer[25710]: [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x837c13]\nJan 23 21:22:12 debian discern-proc-sorcerer[25710]: goroutine 10 [running]:\nJan 23 21:22:12 debian discern-proc-sorcerer[25710]: main.SaveTopCpuProcs({0xc000245f80, 0x5, 0xc00006cb40?})\nJan 23 21:22:12 debian discern-proc-sorcerer[25710]: #011/builds/mergetb/tech/instrumentation/DataSorcerers/cmd/metadata/proc/save.go:108 +0x293\nJan 23 21:22:12 debian discern-proc-sorcerer[25710]: main.main.func1()\nJan 23 21:22:12 debian discern-proc-sorcerer[25710]: #011/builds/mergetb/tech/instrumentation/DataSorcerers/cmd/metadata/proc/main.go:193 +0x18\nJan 23 21:22:12 debian discern-proc-sorcerer[25710]: created by main.main in goroutine 1\nJan 23 21:22:12 debian discern-proc-sorcerer[25710]: #011/builds/mergetb/tech/instrumentation/DataSorcerers/cmd/metadata/proc/main.go:190 +0x2a8\nJan 23 21:22:12 debian systemd[1]: discern-proc.service: Main process exited, code=exited, status=2/INVALIDARGUMENT\nJan 23 21:22:12 debian systemd[1]: discern-proc.service: Failed with result 'exit-code'.\nJan 23 21:22:12 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:22:12] [Ansible] saving config specified in env\nJan 23 21:22:12 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:22:12] [Ansible] saving config and playbooks from home\nJan 23 21:22:12 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:22:12] [Ansible] saving config from ~ and /etc\nJan 23 21:22:12 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:22:12] [Ansible] finished reading. looping\nJan 23 21:22:12 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:22:12] [Ansible] Saving playbooks from /etc/ansible\nJan 23 21:22:12 debian discern-ansible-sorcerer[2673]: #033[31mERRO#033[0m[21:22:12] [Ansible] error reading /etc/ansible: stat /etc/ansible: no such file or directory\nJan 23 21:22:13 debian discern-jupyter-sorcerer[2789]: #033[36mINFO#033[0m[21:22:13] [Jupyter] File search & saving stopped\nJan 23 21:22:13 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:13] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:22:13.507403727Z", "Location":"/var/log/auth.log", "Content":"Jan 23 21:22:11 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:12 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/[ -e /tmp/count.blankcanvas ]\nJan 23 21:22:12 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:22:12 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:12 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/cat /tmp/count.blankcanvas\nJan 23 21:22:12 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:22:12 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:12 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/tee /tmp/count.blankcanvas\nJan 23 21:22:12 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:22:12 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:12 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/touch /var/log/ttylog/ttylog.cache.infra.real.dnsmitm.blankcanvas.blankcanvas.172.trace\nJan 23 21:22:12 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:22:12 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:12 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/chmod ugo+rw /var/log/ttylog/ttylog.cache.infra.real.dnsmitm.blankcanvas.blankcanvas.172.trace\nJan 23 21:22:12 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:22:12 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:12 debian sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/systemctl stop named\nJan 23 21:22:12 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 23 21:22:12 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:12 debian sshd[26368]: Received disconnect from 192.168.254.2 port 58754:11: disconnected by user\nJan 23 21:22:12 debian sshd[26368]: Disconnected from user blankcanvas 192.168.254.2 port 58754\nJan 23 21:22:12 debian sshd[26347]: pam_unix(sshd:session): session closed for user blankcanvas\nJan 23 21:22:12 debian systemd-logind[388]: Session 213 logged out. Waiting for processes to exit.\nJan 23 21:22:12 debian systemd-logind[388]: Removed session 213.\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:22:13.511383604Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:22:12 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:12] [Proc] error stipping top cpu pid 26347: error loading /proc/26347/status: open /proc/26347/status: no such file or directory\nJan 23 21:22:12 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:12] [Proc] error saving proc-cpu info for pid 26350: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:22:12 debian discern-proc-sorcerer[25710]: #033[31mERRO#033[0m[21:22:12] [Proc] error saving proc-cpu info for pid 25710: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: i/o timeout\"\nJan 23 21:22:12 debian discern-proc-sorcerer[25710]: panic: runtime error: invalid memory address or nil pointer dereference\nJan 23 21:22:12 debian discern-proc-sorcerer[25710]: [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x837c13]\nJan 23 21:22:12 debian discern-proc-sorcerer[25710]: goroutine 10 [running]:\nJan 23 21:22:12 debian discern-proc-sorcerer[25710]: main.SaveTopCpuProcs({0xc000245f80, 0x5, 0xc00006cb40?})\nJan 23 21:22:12 debian discern-proc-sorcerer[25710]: #011/builds/mergetb/tech/instrumentation/DataSorcerers/cmd/metadata/proc/save.go:108 +0x293\nJan 23 21:22:12 debian discern-proc-sorcerer[25710]: main.main.func1()\nJan 23 21:22:12 debian discern-proc-sorcerer[25710]: #011/builds/mergetb/tech/instrumentation/DataSorcerers/cmd/metadata/proc/main.go:193 +0x18\nJan 23 21:22:12 debian discern-proc-sorcerer[25710]: created by main.main in goroutine 1\nJan 23 21:22:12 debian discern-proc-sorcerer[25710]: #011/builds/mergetb/tech/instrumentation/DataSorcerers/cmd/metadata/proc/main.go:190 +0x2a8\nJan 23 21:22:12 debian systemd[1]: discern-proc.service: Main process exited, code=exited, status=2/INVALIDARGUMENT\nJan 23 21:22:12 debian systemd[1]: discern-proc.service: Failed with result 'exit-code'.\nJan 23 21:22:12 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:22:12] [Ansible] saving config specified in env\nJan 23 21:22:12 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:22:12] [Ansible] saving config and playbooks from home\nJan 23 21:22:12 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:22:12] [Ansible] saving config from ~ and /etc\nJan 23 21:22:12 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:22:12] [Ansible] finished reading. looping\nJan 23 21:22:12 debian discern-ansible-sorcerer[2673]: #033[36mINFO#033[0m[21:22:12] [Ansible] Saving playbooks from /etc/ansible\nJan 23 21:22:12 debian discern-ansible-sorcerer[2673]: #033[31mERRO#033[0m[21:22:12] [Ansible] error reading /etc/ansible: stat /etc/ansible: no such file or directory\nJan 23 21:22:13 debian discern-jupyter-sorcerer[2789]: #033[36mINFO#033[0m[21:22:13] [Jupyter] File search & saving stopped\nJan 23 21:22:13 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:13] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 23 21:22:13 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:22:13] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:13 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:22:13] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:22:18.524383707Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-86.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-87.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-88.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-89.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-9.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-90.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-91.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-92.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-93.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-94.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 391.\nJan 23 21:22:17 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:17 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:17 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-95.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-96.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-97.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-98.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-99.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-root-0.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-root-1.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:22:17] [Bash] finished sending logs. pausing...\nJan 23 21:22:18 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:22:18] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:18 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:18] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:22:18.530584280Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-87.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-88.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-89.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-9.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-90.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-91.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-92.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-93.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-94.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 391.\nJan 23 21:22:17 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:17 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:17 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-95.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-96.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-97.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-98.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-blankcanvas-99.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-root-0.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[31mERRO#033[0m[21:22:17] [Bash] Error saving cache-root-1.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 23 21:22:17 debian discern-bash-sorcerer[9394]: #033[36mINFO#033[0m[21:22:17] [Bash] finished sending logs. pausing...\nJan 23 21:22:18 debian discern-network-sorcerer[2829]: #033[31mERRO#033[0m[21:22:18] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:18 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:18] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 23 21:22:18 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:22:18] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:22:23.544429051Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:22:22 debian systemd[26350]: dirmngr.socket: Succeeded.\nJan 23 21:22:22 debian systemd[26350]: Closed GnuPG network certificate management daemon.\nJan 23 21:22:22 debian systemd[26350]: gpg-agent-browser.socket: Succeeded.\nJan 23 21:22:22 debian systemd[26350]: Closed GnuPG cryptographic agent and passphrase cache (access for web browsers).\nJan 23 21:22:22 debian systemd[26350]: gpg-agent-extra.socket: Succeeded.\nJan 23 21:22:22 debian systemd[26350]: Closed GnuPG cryptographic agent and passphrase cache (restricted).\nJan 23 21:22:22 debian systemd[26350]: gpg-agent-ssh.socket: Succeeded.\nJan 23 21:22:22 debian systemd[26350]: Closed GnuPG cryptographic agent (ssh-agent emulation).\nJan 23 21:22:22 debian systemd[26350]: gpg-agent.socket: Succeeded.\nJan 23 21:22:22 debian systemd[26350]: Closed GnuPG cryptographic agent and passphrase cache.\nJan 23 21:22:22 debian systemd[26350]: Removed slice User Application Slice.\nJan 23 21:22:22 debian systemd[26350]: Reached target Shutdown.\nJan 23 21:22:22 debian systemd[26350]: systemd-exit.service: Succeeded.\nJan 23 21:22:22 debian systemd[26350]: Finished Exit the Session.\nJan 23 21:22:22 debian systemd[26350]: Reached target Exit the Session.\nJan 23 21:22:22 debian systemd[1]: Stopping User Manager for UID 1001...\nJan 23 21:22:22 debian systemd[1]: user@1001.service: Succeeded.\nJan 23 21:22:22 debian systemd[1]: Stopped User Manager for UID 1001.\nJan 23 21:22:22 debian systemd[1]: Stopping User Runtime Directory /run/user/1001...\nJan 23 21:22:22 debian systemd[1]: run-user-1001.mount: Succeeded.\nJan 23 21:22:22 debian systemd[1]: user-runtime-dir@1001.service: Succeeded.\nJan 23 21:22:22 debian systemd[1]: Stopped User Runtime Directory /run/user/1001.\nJan 23 21:22:22 debian systemd[1]: Removed slice User Slice of UID 1001.\nJan 23 21:22:23 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:23] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:22:23.545930361Z", "Location":"/var/log/auth.log", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:22:23.549968229Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:22:22 debian systemd[26350]: gpg-agent-browser.socket: Succeeded.\nJan 23 21:22:22 debian systemd[26350]: Closed GnuPG cryptographic agent and passphrase cache (access for web browsers).\nJan 23 21:22:22 debian systemd[26350]: gpg-agent-extra.socket: Succeeded.\nJan 23 21:22:22 debian systemd[26350]: Closed GnuPG cryptographic agent and passphrase cache (restricted).\nJan 23 21:22:22 debian systemd[26350]: gpg-agent-ssh.socket: Succeeded.\nJan 23 21:22:22 debian systemd[26350]: Closed GnuPG cryptographic agent (ssh-agent emulation).\nJan 23 21:22:22 debian systemd[26350]: gpg-agent.socket: Succeeded.\nJan 23 21:22:22 debian systemd[26350]: Closed GnuPG cryptographic agent and passphrase cache.\nJan 23 21:22:22 debian systemd[26350]: Removed slice User Application Slice.\nJan 23 21:22:22 debian systemd[26350]: Reached target Shutdown.\nJan 23 21:22:22 debian systemd[26350]: systemd-exit.service: Succeeded.\nJan 23 21:22:22 debian systemd[26350]: Finished Exit the Session.\nJan 23 21:22:22 debian systemd[26350]: Reached target Exit the Session.\nJan 23 21:22:22 debian systemd[1]: Stopping User Manager for UID 1001...\nJan 23 21:22:22 debian systemd[1]: user@1001.service: Succeeded.\nJan 23 21:22:22 debian systemd[1]: Stopped User Manager for UID 1001.\nJan 23 21:22:22 debian systemd[1]: Stopping User Runtime Directory /run/user/1001...\nJan 23 21:22:22 debian systemd[1]: run-user-1001.mount: Succeeded.\nJan 23 21:22:22 debian systemd[1]: user-runtime-dir@1001.service: Succeeded.\nJan 23 21:22:22 debian systemd[1]: Stopped User Runtime Directory /run/user/1001.\nJan 23 21:22:22 debian systemd[1]: Removed slice User Slice of UID 1001.\nJan 23 21:22:23 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:23] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 23 21:22:23 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:22:23] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:23 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:22:23] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:22:28.563851576Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:22:23 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:22:23] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:23 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:22:23] [Logs] error saving /var/log/daemon.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:23 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:23] [Logs] Sending has completed\nJan 23 21:22:24 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 393.\nJan 23 21:22:24 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:24 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:24 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:22:25 debian discern-proc-sorcerer[26493]: #033[36mINFO#033[0m[21:22:25] [Proc] successfully saved data to the fusion core in normal loop\nJan 23 21:22:25 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:25] [Proc] error saving proc-cpu info for pid 26493: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:25 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:25] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:25 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:25] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:25 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:25] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:25 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:25] [Proc] error saving proc-cpu info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:25 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:25] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:25 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:25] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:25 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:25] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:25 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:25] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:25 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:25] [Proc] error saving proc-mem info for pid 9394: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:27 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 394.\nJan 23 21:22:27 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:27 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:27 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:22:28 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:28] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:22:28.565364166Z", "Location":"/var/log/auth.log", "Content":"Jan 23 21:22:24 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/fork.bt\nJan 23 21:22:24 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/execveat.bt\nJan 23 21:22:24 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/openat2.bt\nJan 23 21:22:24 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/execve.bt\nJan 23 21:22:24 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:24 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:24 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/kill.bt\nJan 23 21:22:24 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/openat.bt\nJan 23 21:22:24 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:24 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:24 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:24 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:24 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:24 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:24 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:24 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:24 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:24 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:24 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:24 debian sudo: pam_unix(sudo:session): session closed for user root\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:22:28.569444045Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:22:23 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:23] [Logs] Sending has completed\nJan 23 21:22:24 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 393.\nJan 23 21:22:24 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:24 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:24 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:22:25 debian discern-proc-sorcerer[26493]: #033[36mINFO#033[0m[21:22:25] [Proc] successfully saved data to the fusion core in normal loop\nJan 23 21:22:25 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:25] [Proc] error saving proc-cpu info for pid 26493: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:25 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:25] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:25 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:25] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:25 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:25] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:25 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:25] [Proc] error saving proc-cpu info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:25 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:25] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:25 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:25] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:25 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:25] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:25 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:25] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:25 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:25] [Proc] error saving proc-mem info for pid 9394: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:27 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 394.\nJan 23 21:22:27 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:27 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:27 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:22:28 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:28] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 23 21:22:28 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:22:28] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:28 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:22:28] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:22:33.583559377Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:22:28 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:22:28] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:28 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:22:28] [Logs] error saving /var/log/daemon.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:28 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:28] [Logs] Sending has completed\nJan 23 21:22:30 debian discern-proc-sorcerer[26493]: #033[36mINFO#033[0m[21:22:30] [Proc] successfully saved data to the fusion core in normal loop\nJan 23 21:22:30 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 395.\nJan 23 21:22:30 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:30 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:30 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:22:30 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:30] [Proc] error saving proc-cpu info for pid 26493: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:30 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:30] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:30 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:30] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:30 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:30] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:30 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:30] [Proc] error saving proc-cpu info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:31 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:31] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:31 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:31] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:31 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:31] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:31 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:31] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:31 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:31] [Proc] error saving proc-mem info for pid 9394: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:32 debian discern-network-sorcerer[2829]: #033[36mINFO#033[0m[21:22:32] [Network] saved network slice\nJan 23 21:22:33 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:33] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:22:33.585026447Z", "Location":"/var/log/auth.log", "Content":"Jan 23 21:22:30 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:30 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:30 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/open.bt\nJan 23 21:22:30 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:30 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:30 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/openat2.bt\nJan 23 21:22:30 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:30 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:30 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/recvmmsg.bt\nJan 23 21:22:30 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:30 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:30 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/openat.bt\nJan 23 21:22:30 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:30 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/fork.bt\nJan 23 21:22:30 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:30 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:30 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:30 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:30 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:30 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:30 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:30 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/recvfrom.bt\nJan 23 21:22:30 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:30 debian sudo: pam_unix(sudo:session): session closed for user root\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:22:33.589099045Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:22:28 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:28] [Logs] Sending has completed\nJan 23 21:22:30 debian discern-proc-sorcerer[26493]: #033[36mINFO#033[0m[21:22:30] [Proc] successfully saved data to the fusion core in normal loop\nJan 23 21:22:30 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 395.\nJan 23 21:22:30 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:30 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:30 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:22:30 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:30] [Proc] error saving proc-cpu info for pid 26493: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:30 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:30] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:30 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:30] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:30 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:30] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:30 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:30] [Proc] error saving proc-cpu info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:31 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:31] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:31 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:31] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:31 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:31] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:31 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:31] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:31 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:31] [Proc] error saving proc-mem info for pid 9394: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:32 debian discern-network-sorcerer[2829]: #033[36mINFO#033[0m[21:22:32] [Network] saved network slice\nJan 23 21:22:33 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:33] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 23 21:22:33 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:22:33] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:33 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:22:33] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:22:38.602980206Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:22:33 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:22:33] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:33 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:22:33] [Logs] error saving /var/log/daemon.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:33 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:33] [Logs] Sending has completed\nJan 23 21:22:34 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 396.\nJan 23 21:22:34 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:34 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:34 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:22:35 debian discern-proc-sorcerer[26493]: #033[36mINFO#033[0m[21:22:35] [Proc] successfully saved data to the fusion core in normal loop\nJan 23 21:22:36 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:36] [Proc] error saving proc-cpu info for pid 26493: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:36 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:36] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:36 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:36] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:36 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:36] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:36 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:36] [Proc] error saving proc-cpu info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:36 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:36] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:36 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:36] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:36 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:36] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:36 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:36] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:36 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:36] [Proc] error saving proc-mem info for pid 26493: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:37 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 397.\nJan 23 21:22:37 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:37 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:37 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:22:38 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:38] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:22:38.604556677Z", "Location":"/var/log/auth.log", "Content":"Jan 23 21:22:37 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/open.bt\nJan 23 21:22:37 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/fork.bt\nJan 23 21:22:37 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:37 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:37 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:37 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:37 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:37 debian sudo: pam_unix(sudo:session): session closed for user root\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:22:38.608721296Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:22:33 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:33] [Logs] Sending has completed\nJan 23 21:22:34 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 396.\nJan 23 21:22:34 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:34 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:34 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:22:35 debian discern-proc-sorcerer[26493]: #033[36mINFO#033[0m[21:22:35] [Proc] successfully saved data to the fusion core in normal loop\nJan 23 21:22:36 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:36] [Proc] error saving proc-cpu info for pid 26493: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:36 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:36] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:36 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:36] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:36 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:36] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:36 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:36] [Proc] error saving proc-cpu info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:36 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:36] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:36 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:36] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:36 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:36] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:36 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:36] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:36 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:36] [Proc] error saving proc-mem info for pid 26493: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:37 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 397.\nJan 23 21:22:37 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:37 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:37 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:22:38 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:38] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 23 21:22:38 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:22:38] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:38 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:22:38] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:22:43.623226100Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:22:38 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:22:38] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:38 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:22:38] [Logs] error saving /var/log/daemon.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:38 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:38] [Logs] Sending has completed\nJan 23 21:22:40 debian discern-proc-sorcerer[26493]: #033[36mINFO#033[0m[21:22:40] [Proc] successfully saved data to the fusion core in normal loop\nJan 23 21:22:40 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 398.\nJan 23 21:22:40 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:40 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:40 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:22:41 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:41] [Proc] error saving proc-cpu info for pid 26493: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:41 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:41] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:41 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:41] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:41 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:41] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:41 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:41] [Proc] error saving proc-cpu info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:41 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:41] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:41 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:41] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:41 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:41] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:41 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:41] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:41 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:41] [Proc] error saving proc-mem info for pid 26493: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:41 debian discern-network-sorcerer[2829]: #033[36mINFO#033[0m[21:22:41] [Network] saved network slice\nJan 23 21:22:43 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:43] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:22:43.628441607Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:22:38 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:38] [Logs] Sending has completed\nJan 23 21:22:40 debian discern-proc-sorcerer[26493]: #033[36mINFO#033[0m[21:22:40] [Proc] successfully saved data to the fusion core in normal loop\nJan 23 21:22:40 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 398.\nJan 23 21:22:40 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:40 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:40 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:22:41 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:41] [Proc] error saving proc-cpu info for pid 26493: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:41 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:41] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:41 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:41] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:41 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:41] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:41 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:41] [Proc] error saving proc-cpu info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:41 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:41] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:41 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:41] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:41 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:41] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:41 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:41] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:41 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:41] [Proc] error saving proc-mem info for pid 26493: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:41 debian discern-network-sorcerer[2829]: #033[36mINFO#033[0m[21:22:41] [Network] saved network slice\nJan 23 21:22:43 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:43] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 23 21:22:43 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:22:43] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:22:48.641324233Z", "Location":"/var/log/syslog", "Content":"Jan 23 21:22:43 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:22:43] [Logs] error saving /var/log/daemon.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:43 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:43] [Logs] Sending has completed\nJan 23 21:22:43 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 399.\nJan 23 21:22:43 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:43 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:43 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:22:45 debian discern-proc-sorcerer[26493]: #033[36mINFO#033[0m[21:22:45] [Proc] successfully saved data to the fusion core in normal loop\nJan 23 21:22:46 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:46] [Proc] error saving proc-cpu info for pid 26493: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:46 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:46] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:46 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:46] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:46 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:46] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:46 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:46] [Proc] error saving proc-cpu info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:46 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:46] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:46 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:46] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:46 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:46] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:46 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:46] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:46 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:46] [Proc] error saving proc-mem info for pid 26493: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:47 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 400.\nJan 23 21:22:47 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:47 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:47 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:22:48 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:48] [Logs] /var/log/dmesg not found. trying to capture from cli\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:22:48.642780232Z", "Location":"/var/log/auth.log", "Content":"Jan 23 21:22:43 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/open.bt\nJan 23 21:22:43 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/openat.bt\nJan 23 21:22:43 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/execve.bt\nJan 23 21:22:43 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/openat2.bt\nJan 23 21:22:43 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:43 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:43 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/execveat.bt\nJan 23 21:22:43 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:43 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:43 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:43 debian sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/kill.bt\nJan 23 21:22:43 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:43 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:43 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:43 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:43 debian sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 23 21:22:43 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:43 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:43 debian sudo: pam_unix(sudo:session): session closed for user root\nJan 23 21:22:43 debian sudo: pam_unix(sudo:session): session closed for user root\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
{"TimeStamp":"2025-01-23T21:22:48.647887778Z", "Location":"/var/log/daemon.log", "Content":"Jan 23 21:22:43 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:43] [Logs] Sending has completed\nJan 23 21:22:43 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 399.\nJan 23 21:22:43 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:43 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:43 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:22:45 debian discern-proc-sorcerer[26493]: #033[36mINFO#033[0m[21:22:45] [Proc] successfully saved data to the fusion core in normal loop\nJan 23 21:22:46 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:46] [Proc] error saving proc-cpu info for pid 26493: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:46 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:46] [Proc] error saving proc-cpu info for pid 1: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:46 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:46] [Proc] error saving proc-cpu info for pid 140: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:46 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:46] [Proc] error saving proc-cpu info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:46 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:46] [Proc] error saving proc-cpu info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:46 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:46] [Proc] error saving proc-mem info for pid 179: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:46 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:46] [Proc] error saving proc-mem info for pid 2829: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:46 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:46] [Proc] error saving proc-mem info for pid 380: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:46 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:46] [Proc] error saving proc-mem info for pid 2808: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:46 debian discern-proc-sorcerer[26493]: #033[31mERRO#033[0m[21:22:46] [Proc] error saving proc-mem info for pid 26493: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:47 debian systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 400.\nJan 23 21:22:47 debian systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:47 debian systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 23 21:22:47 debian systemd[1]: discern-os-record.service: Succeeded.\nJan 23 21:22:48 debian discern-logs-sorcerer[2808]: #033[36mINFO#033[0m[21:22:48] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 23 21:22:48 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:22:48] [Logs] error saving /var/log/syslog: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\nJan 23 21:22:48 debian discern-logs-sorcerer[2808]: #033[31mERRO#033[0m[21:22:48] [Logs] error saving /var/log/auth.log: client error saving log to fusion core: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp: lookup fusioncore on 10.1.2.3:53: no such host\"\n", "DevID":"cache.infra.real.dnsmitm.blankcanvas"}
